tests/testthat/test-get_table.R

```diff
@@ -64,3 +64,9 @@ test_that("get_table() works", { for (conn in conns) { # nolint: brace_linter
   t <- id("tset.mtcars", conn)
   expect_error(get_table(conn, t), regexp = "Table tset.mtcars is not found!")
 }})
+
+test_that("get_table returns list of tables if no table is requested", { for (conn in conns) {
```

Comment:
I think this will also give a linting error. Check and add # nolint: brace_linter if needed